### PR TITLE
Added a note about "systemct enable" and cloud providers

### PR DESCRIPTION
**Pull request by kayrus**
Resolves https://github.com/coreos/docs/issues/722

/cc @joshix @robszumski

### CODE REVIEW

File: os/getting-started-with-systemd.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ systemd consists of two main concepts: a unit and a target. A unit is a configur
 
 systemd is the first process started on CoreOS and it reads different targets and starts the processes specified which allows the operating system to start. The target that you'll interact with is the `multi-user.target` which holds all of the general use unit files for our containers.
 
-Each target is actually a collection of symlinks to our unit files. This is specified in the unit file by `WantedBy=multi-user.target`. Running `systemctl enable foo.service` creates symlinks to the unit inside `multi-user.target.wants`.
+Each target is actually a collection of symlinks to our unit files. This is specified in the unit file by `WantedBy=multi-user.target`. Running `systemctl enable foo.service` creates symlinks to the unit inside `multi-user.target.wants`. Please note that `systemctl enable foo.service` may affect the boot process by race condition exception in network configuration when [Cloud-Config] configures network according to cloud provider's `meta_data` and `user_data`. For the common usage you have to put your unit files inside Cloud-Config's [units][cloud-config-units] section.
```
Comment (robszumski) [inline os/getting-started-with-systemd.md]: This sentence is really hard to parse/understand, but I don't really have ideas for how to rewrite. @joshix?

**Comment by kayrus** [comment]
These `systemct enable %service%` issues are quite common. This week I already met two users with this problem.

 We have to aware them.

**Comment by kayrus** [comment]
Probably we have to add this info at Cloud-Config doc here: https://github.com/coreos/coreos-cloudinit/blob/master/Documentation/cloud-config.md#units

File: os/getting-started-with-systemd.md (hunk 1 above)
Comment (crawford) [inline os/getting-started-with-systemd.md]: I don't like this advice. I think we should tell our users that if they want to use cloud-init, all services should be `runtime: true` and `enabled: false`. This way, every boot is the same.

There is nothing wrong with systemd enable, it is entirely our fault. When people start moving to Ignition, they will use systemd enable to run there services and it will work perfectly. I don't want to mislead people here.

**Comment by joshix** [comment]
closed inactive and addressed otherwise in both cloud-config.md and with ignition switch
